lib/ripper-tags.rb

```diff
@@ -109,6 +109,9 @@ def self.option_parser(options)
       opts.on("-n", "Equivalent to --excmd=number.") do
         options.excmd = "number"
       end
+      opts.on("--fields=+l", "Include language information in the tag") do |flags|
```

Comment:
There's already a block that processes the `--fields` option: `opts.on("--fields=+n"`. There doesn't have to be a new one. You can change its signature to `opts.on("--fields=+nl"` (to demonstrate the two flags that we support) and change its description to something like “add extra fields to output; see `ctags --list-fields` for explanation”.

Reply:
Good point! I wrongly assumed that `opts.on`'s first argument was an exact match thing.